src/Resources/Custom TypeEditor Template.md

```diff
@@ -0,0 +1,314 @@
+Custom TypeEditor Template
+
+Since .NET 3.1 started to support the WinForms Runtime, a new WinForms designer
+was needed to support .NET applications. The work required a near-complete
+rearchitecting of the designer, as we responded to the differences between .NET
+and the .NET Framework based WinForms designer everyone knows and loves.
+
+Until we added support for .NET Core applications there was only a single
+process, devenv.exe, that both the Visual Studio environment and the application
+being designed ran within. But .NET Framework and .NET Core can’t both run
+together within devenv.exe, and as a result we had to take the designer out of
+process, thus we called the new designer – WinForms Out of Process Designer (or
+OOP designer for short). We call the existing process – the process that Visual
+Studio runs in – the client process, and the process which shows the actual Form
+at Design time, the server process, which we call *DesignToolsServer*.
+
+# Enter the DesignToolsServer
+
+While simple Control Designer scenarios like Type Converters, Action Lists or
+CodeDom Serializers don’t need any substantial rewrites, Type Editors are a
+different beast altogether.
```

Comment:
Unnecessary capitalisation throughout. I'm not going to comment on every instance.
```suggestion
While simple control designer scenarios that use type converters, action lists or
CodeDom serializers may not need any substantial changes, designers that use type editors require significantly more authoring effort.
```